lib/active_merchant/billing/gateways/payu_latam.rb

```diff
@@ -236,6 +253,9 @@ def add_payer(post, options)
           payer[:fullName] = address[:name]
           payer[:contactPhone] = address[:phone]
           payer[:dniNumber] = options[:dni_number]
+          payer[:dniType] = options[:dni_type]
```

Comment:
Should this have a guard like we do in https://github.com/deedeelavinder/active_merchant/blob/dbd6f141c6ae0e3db15d3b3f37d0bf50f5c79663/lib/active_merchant/billing/gateways/payu_latam.rb#L163?